.travis.yml

```diff
@@ -57,9 +57,9 @@ jobs:
         - rm -rf $HOME/.ivy2
         - cd $TRAVIS_BUILD_DIR && sbt clean stage
         # only uncomment if docker image is needed for this branch. remove before merging into develop
-        # - docker login -u $DOCKER_USER -p $DOCKER_PASS
-        # - sbt webapi/docker:publish
-        # - sbt salsah1/docker:publish
+        - docker login -u $DOCKER_USER -p $DOCKER_PASS
```

Comment:
@subotic I am about to merge this branch. Ok for you?

Reply:
I undid the changes in 
https://github.com/dhlab-basel/Knora/pull/890/commits/32a1ef3a4dda541fcc303386890e93fa6679bbb9